### DISCUSSION TITLE
docs/config/translate

### DISCUSSION THREAD

**Discussion by giscus** [STranslate/STranslate#141] [Announcements]
# docs/config/translate

WPF 开发的一款即开即用、即用即走的翻译(OCR)工具

https://stranslate.zggsong.com/docs/config/translate.html

**Comment by xinxuezi** [comment]
你好，这个请问一下，这个出现
请求超时: The request was canceled due to the configured HttpClient.Timeout of 10 seconds elapsing.
该怎么办吗？

**Reply by ZGGSONG** [reply]
涵义：请求默认超时10s，超时后将取消请求，一般来说服务10还没响应就说明你的网络环境无法访问该服务或者说该服务本身就有问题。
解决：从你issue来看，那两个免费服务我跑在deno上的，服务器本身就在国外，国内有的地方无法访问，一般来说，要么代理，要么配置其他服务。

**Reply by wangchengyu** [reply]
希望作者可以加入一个配置项目可以修改网络访问超时时间，或者将其加长一点。我本地启动ollama模型加载超过了10s，除非我手动用命令提前加载模型。
感谢作者做了这么好用的软件，比deepL的桌面端强太多了[赞]。

**Reply by ZGGSONG** [reply]
> 希望作者可以加入一个配置项目可以修改网络访问超时时间，或者将其加长一点。我本地启动ollama模型加载超过了10s，除非我手动用命令提前加载模型。
> 感谢作者做了这么好用的软件，比deepL的桌面端强太多了[赞]。

ok，去提一个feature吧，怕忘了

**Comment by Skywalker144** [comment]
使用OCR时报错：“OCR失败: 离线数据不完整”如何解决呢

**Reply by Skywalker144** [reply]
解决了

**Comment by songjiang798** [comment]
你好，我有openai的api。请问怎么接入这个软件里面呢？

**Reply by ZGGSONG** [reply]
请到设置里随便翻翻呢，各项功能都写的很清楚

**Reply by songjiang798** [reply]
那个接口怎么弄哦？能教教吗？

**Reply by ZGGSONG** [reply]
设置-翻译服务-左下角有个加号-添加服务-openai 填api和key就行

**Reply by songjiang798** [reply]
api和key不是同一个码吗？就是那个sk开头的。不好意思。，我刚刚接触这个。

**Reply by songjiang798** [reply]
有飞机吗哥？教教我

**Comment by songjiang798** [comment]
OCR出错: Unable to load DLL 'PaddleOCR.dll' or one of its dependencies: 找不到指定的模块。 (0x8007007E)
這個是什麼原因呀？

**Reply by ZGGSONG** [reply]
这个可能是因为底层c++依赖缺失，请前往paddleocrsharp项目查找

https://gitee.com/raoyutian/PaddleOCRSharp/blob/master/doc/README_question.md

**Comment by IvanRJ** [comment]
你好，請問一下 Stranslate 服務遇到
翻译出错: '0x1F' is an invalid start of a value. Path: $ | LineNumber: 0 | BytePositionInLine: 0.
這個要怎麼修？

**Reply by ZGGSONG** [reply]
> 你好，請問一下 Stranslate 服務遇到
> 翻译出错: '0x1F' is an invalid start of a value. Path: $ | LineNumber: 0 | BytePositionInLine: 0.
> 這個要怎麼修？

#236

**Comment by Maotiannan** [comment]
有了DPPL的API KEY，但是输入https://api-free.deepl.com和key后验证失败，已经在沉浸式翻译中用了是可以用的这个key，是怎么回事呢

**Reply by ZGGSONG** [reply]
请问添加的是deepl官方服务嘛

**Reply by Maotiannan** [reply]
key我是在官方申请的，我有信用卡，然后添加的服务设置-文本翻译-DeepLX(自建)，然后接口用的https://api-free.deepl.com，填入key后点击验证就是验证失败

**Reply by ZGGSONG** [reply]
服务列表里每个服务右边有标签，请添加deepl官方服务再进行配置

**Reply by Maotiannan** [reply]
谢谢，对上了这次！

**Comment by thee6688** [comment]
gemini是不是地址有问题，填入api key还是测试失败，请问下，还需要设置哪些吗？

**Reply by ZGGSONG** [reply]
谷歌的服务被国内屏蔽了，如果是网络问题请自行解决

**Reply by thee6688** [reply]
感谢回复，我是泰国，应该不是网络问题

**Reply by ZGGSONG** [reply]
你换个模型试试呢，目前我每天也在用gemini。

**Reply by thee6688** [reply]
请问如何更换？我就是卡在这里，我目前使用的就是软件中的服务+key，一直验证失败。。。

**Reply by ZGGSONG** [reply]
![图片](https://github.com/user-attachments/assets/6868be5a-9cec-421f-acfb-935047d807a7)

**Comment by fugubiao** [comment]
大佬，ollama 使用deepseek 模型，前期<think>...</think>内会有很多自言自语，能不能可选将大模型<think>...</think>自言自语过滤掉呢？

**Comment by xiangxiangji** [comment]
大佬 我的openai接入的key是sk-proj 开头 一直验证失败 可是我已经调成个人key了 怎么解决啊

**Comment by WMdoit** [comment]
请问stranslate编译生成时怎么把大小控制在100m，我用vs2022默认的编译生成，有900m，和release的大小差别好大

**Reply by WMdoit** [reply]
和大佬你生成的对比哈，我的release有100m，runtimes文件夹里有大几百m，不知道为什么

**Comment by xiangxiangji** [comment]
发错人了兄弟

王明 ***@***.***>于2025年9月5日 周五09:53写道：

**Reply by WMdoit** [reply]
大佬，帮我解决下把

**Comment by Teneurin-3** [comment]
请问内置的必应词典和金山词霸为何失效了呢？stranslate就能使用，但是没有相关字典和音标，不太方便

**Reply by CATBirdlong** [reply]
问题+1, 内置的bingdict和kingsoftdict均会失效, 不定报错服务无法访问或超时, 大概率不响应, azure疑似是回收功能和接口, 金山不明.
